pkg/planner/core/casetest/instanceplancache/builtin_func_test.go

```diff
@@ -0,0 +1,325 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instanceplancache
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"testing"
+
+	"github.com/pingcap/tidb/pkg/testkit"
+)
+
+func TestBuiltinInIntSig(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec(`set global tidb_enable_instance_plan_cache=1`)
```

Comment:
Do we need to reset this var in defer

Reply:
Good suggestion but this seems unnecessary since we're using Bazel and it runs each package seperately, and all test cases in this package "instanceplancache" would set "tidb_enable_instance_plan_cache=1".